docs/trust/security/pci-dss.md

```diff
@@ -0,0 +1,70 @@
+
+Our **Responsibility Matrix** indicated which parts of the SAQ A requirements are handled by us and which are to be done by you.
+We handle most of the technical requirements, but for example requirements 12.8.1 - 12.8.5 require you to keep a list of all service providers (such as us) and regularly check that we are still PCI DSS compliant.
+
+You can find the latest version of all relevant documents, such as our **Attestation of Scan Compliance**, our **Attestation of Compliance** and the **Responsibility Matrix** for our relationship with you in your pretix account at :navpath:Your organizer → :fa3-wrench: Settings → PCI DSS compliance:.
+
+Should you require any additional documentation or have any further questions on the topic, please reach out to us at [pci@pretix.eu](mailto:support@pretix.eu).
+
+## Self-hosted versions of pretix
+
+<!-- md:community -->
+<!-- md:enterprise -->
+
+If you host pretix on your own servers, you need to ensure compliance with **all requirements** listed in the SAQ A questionnaire.
+This includes technical requirements such as using strong authentication methods, process requirements such as ensuring you always apply security updates quickly and having an incident response plan, as well as the requirement to perform **quarterly vulnerability scans** on your server through a PCI Approved Scanning Vendor (ASV).
+
+The scan should at the very least include all servers used to run pretix, as well as possibly your website embeding our [Widget](../../guides/widget.md).
+The later part does not apply when you use a payment provider where the credit card form always opens in a new tab or when you configure the widget to [always open in a new tab](../../guides/widget.md#always-open-a-new-tab).
+There is a list of [Approved Scanning Vendors](https://www.pcisecuritystandards.org/assessors_and_solutions/approved_scanning_vendors/) on the PCI website, such as [usd AG](https://www.usd.de/en/pci-payment-security/pci-security-scans/) which is performing our scans.
+
+### SAQ A eligibility criteria and custom plugins
+
+pretix and official pretix plugins never store any credit card numbers directly, therefore most of the SAQ A eligibility criteria are always true when using pretix with only official plugins.
+If you create a custom payment plugin, you need to make sure that this requirement stays true and all credit card inputs are served directly by a PCI certified payment service provider through an iframe or redirect.
+Implementing payment providers where a HTML form within pretix collects the credit card information would require at least complying with the requirements of the SAQ A-EP questionnaire.
+
+The most recent version of the SAQ A also includes the following requirement:
```

Comment:
Perhaps refer to a specific version or date?

Reply:
```suggestion
Version 4.0.1 of the SAQ A also includes the following requirement:
```